src/evotorch/__init__.py

```diff
@@ -43,6 +43,20 @@
     )
     __version__ = "undefined"
 
+# flake8: noqa
```

Comment:
This is great, it should be helpful for a lot of edge cases. We need to make sure we fully document it.

Reply:
Thanks & agreed. An overall update on docs is planned for soon. While updating the docs, we can mention this feature, as you suggest.